# Generated by Django 4.0.2 on 2022-03-12 07:00

import datetime
from django.conf import settings
from django.db import migrations, models
import django.db.models.deletion
from django.utils.timezone import utc


class Migration(migrations.Migration):

    dependencies = [
        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
        ('challenges', '0003_remove_challenge_modified_alter_challenge_created_and_more'),
    ]

    operations = [
        migrations.AlterField(
            model_name='challenge',
            name='date_finish',
            field=models.DateField(default=datetime.datetime(2022, 3, 19, 7, 0, 2, 183120, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='challenge',
            name='date_start',
            field=models.DateField(default=datetime.datetime(2022, 3, 12, 7, 0, 2, 182988, tzinfo=utc)),
        ),
        migrations.AlterField(
            model_name='photo',
            name='challenge',
            field=models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='challenges.challenge'),
        ),
        migrations.CreateModel(
            name='challenger',
            fields=[
                ('id', models.BigAutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
                ('created', models.DateTimeField(auto_now_add=True)),
                ('updated', models.DateTimeField(auto_now=True)),
                ('challenge', models.ForeignKey(on_delete=django.db.models.deletion.CASCADE, to='challenges.challenge')),
                ('challengers', models.ManyToManyField(blank=True, to=settings.AUTH_USER_MODEL)),
            ],
            options={
                'abstract': False,
            },
        ),
    ]
